index.html

```diff
@@ -52,6 +61,234 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 3-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 4-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 5-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 6-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--about section start-->
+   <section class="card-3">
+    <div class="card-about-title">About me</div>
```

Comment:
- [ ] I think it would be better if you use `<hx>` tags for title of your sections. For more clarity, please visit above comments 👆